server/datastore/mysql/microsoft_mdm.go

```diff
@@ -1562,3 +1560,34 @@ func (ds *Datastore) bulkSetPendingMDMWindowsHostProfilesDB(
 
 	return nil
 }
+
+func (ds *Datastore) GetHostMDMWindowsProfiles(ctx context.Context, hostUUID string) ([]fleet.HostMDMWindowsProfile, error) {
+	stmt := fmt.Sprintf(`
+SELECT
+	profile_uuid,
+	'' AS identifier,
```

Comment:
I think the `fleet.HostMDMWindowsProfile` struct should just not have that `Identifier` field, and the empty value should be filled by `ToHostMDMProfile()`?